packages/components/pie-modal/README.md

```diff
@@ -57,22 +57,22 @@ import { PieModal } from '@justeattakeaway/pie-modal/dist/react';
-| returnFocusAfterCloseSelector | `String` | `undefined` | If provided, focus will be sent to the first element that matches this selector when the modal is closed. If not provided, the `dialog` element will return focus to the element that opened the modal.                                                                                                                                                                                                                                                                                                                                                                               |
-| leadingAction                 | `Object` | `undefined` | An object representing the leading action of the modal. It has the following properties:<br>- text (required): The text to display on the leading action button.<br>- variant: The variant of the leading action button. Please see [pie-button](https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-button/README.md#props) for options. Defaults to primary if not provided.<br>- ariaLabel: The ARIA label for the leading action button. Example: `leadingAction = { text: "Example Text", variant: "secondary", ariaLabel: "Example Aria Label" }`          |
-| supportingAction              | `Object` | `undefined` | An object representing the supporting action of the modal. It has the following properties:<br>- text (required): The text to display on the supporting action button.<br>- variant: The variant of the supporting action button. Please see [pie-button](https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-button/README.md#props) for options. Defaults to ghost if not provided.<br>- ariaLabel: The ARIA label for the supporting action button. Example: `supportingAction = { text: "Example Text", variant: "ghost", ariaLabel: "Example Aria Label" }` |
-| aria                          | `Object` | `undefined` | An object representing the aria labels for the `close` & `back` buttons within the modal as well as the `isLoading` state labels (`aria-label` & `aria-busy`).                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| Property                      | Type      | Default        | Description |
```

Comment:
Just a note – most of these changes are ordering the properties to be alphabetical, so it's easier to scan them (plus adding `hasStackedActions` prop)

Reply:
In the future it would be great to automate these prop tables based off of the `custom-elements.json` file